<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dash</class>
 <widget class="QDialog" name="Dash">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1910</width>
    <height>1060</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-20</y>
     <width>1920</width>
     <height>1080</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(44, 51, 51);</string>
   </property>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>1390</x>
      <y>350</y>
      <width>511</width>
      <height>171</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>SF Pro Rounded</family>
      <pointsize>28</pointsize>
      <weight>10</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
color: rgb(221, 165, 51);
background-color: transparent;
font: 81 28pt &quot;SF Pro Rounded&quot;;</string>
    </property>
    <property name="text">
     <string>Face Analysis &amp; ANPR</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>1400</x>
      <y>40</y>
      <width>411</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>SF Pro Rounded</family>
      <pointsize>28</pointsize>
      <weight>10</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(221, 165, 51);

background-color: transparent;
font: 81 28pt &quot;SF Pro Rounded&quot;;</string>
    </property>
    <property name="text">
     <string>Remote the robot</string>
    </property>
   </widget>
   <widget class="QToolButton" name="bt_4">
    <property name="geometry">
     <rect>
      <x>1550</x>
      <y>200</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(14, 131, 136);
color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>‚óÄÔ∏è</string>
    </property>
   </widget>
   <widget class="QToolButton" name="bt_6">
    <property name="geometry">
     <rect>
      <x>1660</x>
      <y>200</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(14, 131, 136);
color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>‚ñ∂Ô∏è</string>
    </property>
   </widget>
   <widget class="QToolButton" name="bt_8">
    <property name="geometry">
     <rect>
      <x>1600</x>
      <y>150</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(14, 131, 136);
color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>üîº</string>
    </property>
   </widget>
   <widget class="QToolButton" name="bt_2">
    <property name="geometry">
     <rect>
      <x>1600</x>
      <y>250</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(14, 131, 136);
color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>üîΩ</string>
    </property>
   </widget>
   <widget class="QToolButton" name="bt_7">
    <property name="geometry">
     <rect>
      <x>1540</x>
      <y>140</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(14, 131, 136);
color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>‚ÜñÔ∏è</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="bt_9">
    <property name="geometry">
     <rect>
      <x>1670</x>
      <y>140</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(14, 131, 136);
color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>‚ÜóÔ∏è</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="bt_1">
    <property name="geometry">
     <rect>
      <x>1540</x>
      <y>260</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(14, 131, 136);
color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>‚ÜôÔ∏è</string>
    </property>
   </widget>
   <widget class="QToolButton" name="bt_3">
    <property name="geometry">
     <rect>
      <x>1670</x>
      <y>260</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(14, 131, 136);
color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>‚ÜòÔ∏è</string>
    </property>
   </widget>
   <widget class="QToolButton" name="bt_0">
    <property name="geometry">
     <rect>
      <x>1600</x>
      <y>200</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(14, 131, 136);
color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>üîÑ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_anpr">
    <property name="geometry">
     <rect>
      <x>1510</x>
      <y>550</y>
      <width>221</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(221, 165, 51);
color: rgb(74, 37, 0);

font: 81 10pt &quot;SF Pro Rounded&quot;;
</string>
    </property>
    <property name="text">
     <string>Active ANPR</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_face">
    <property name="geometry">
     <rect>
      <x>1420</x>
      <y>860</y>
      <width>201</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(221, 165, 51);
color: rgb(74, 37, 0);

font: 81 10pt &quot;SF Pro Rounded&quot;;
</string>
    </property>
    <property name="text">
     <string>Active Face recognition</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>1440</x>
      <y>590</y>
      <width>371</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>SF Pro Display</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(46, 79, 79);</string>
    </property>
    <property name="title">
     <string>Details</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>201</width>
       <height>71</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="font">
         <font>
          <family>SF Pro Display</family>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>License Number Plate:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_10">
        <property name="font">
         <font>
          <family>SF Pro Display</family>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Status:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>20</y>
       <width>161</width>
       <height>71</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLineEdit" name="lineEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>SF Pro Display</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="frame">
         <bool>false</bool>
        </property>
        <property name="cursorMoveStyle">
         <enum>Qt::LogicalMoveStyle</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="statuslabel">
        <property name="font">
         <font>
          <family>SF Pro Display</family>
          <pointsize>12</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QLabel" name="imgLabel">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>110</y>
      <width>781</width>
      <height>500</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>4</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>800</width>
      <height>500</height>
     </size>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="frameP">
    <property name="geometry">
     <rect>
      <x>1550</x>
      <y>480</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>SF Pro Display</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>ANPR Result</string>
    </property>
   </widget>
   <widget class="QLabel" name="frameF">
    <property name="geometry">
     <rect>
      <x>1420</x>
      <y>740</y>
      <width>201</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>SF Pro Display</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>&quot;unknown&quot; Person</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>1640</x>
      <y>730</y>
      <width>261</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>SF Pro Display</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(46, 79, 79);
</string>
    </property>
    <property name="title">
     <string>Details</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>124</width>
       <height>111</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_11">
        <property name="font">
         <font>
          <family>SF Pro Display</family>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Employee ID:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <family>SF Pro Display</family>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>FirstName : </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <family>SF Pro Display</family>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>LastName:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_4">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>121</width>
       <height>111</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="employeeid">
        <property name="font">
         <font>
          <family>SF Pro Display</family>
          <pointsize>11</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>&quot;unknown&quot;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="firstname">
        <property name="font">
         <font>
          <family>SF Pro Display</family>
          <pointsize>11</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>&quot;unknown&quot;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lastname">
        <property name="font">
         <font>
          <family>SF Pro Display</family>
          <pointsize>11</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>&quot;unknown&quot;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPushButton" name="bt_viewall_2">
    <property name="geometry">
     <rect>
      <x>1700</x>
      <y>940</y>
      <width>90</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(221, 165, 51);
color: rgb(74, 37, 0);

font: 81 10pt &quot;SF Pro Rounded&quot;;
</string>
    </property>
    <property name="text">
     <string>Matching</string>
    </property>
   </widget>
   <widget class="QLabel" name="frameP_2">
    <property name="geometry">
     <rect>
      <x>1410</x>
      <y>900</y>
      <width>241</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>SF Pro Display</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Does the sets match Or not</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_add">
    <property name="geometry">
     <rect>
      <x>1800</x>
      <y>940</y>
      <width>90</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(221, 165, 51);
color: rgb(74, 37, 0);

font: 81 10pt &quot;SF Pro Rounded&quot;;
</string>
    </property>
    <property name="text">
     <string>Alert</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_check">
    <property name="geometry">
     <rect>
      <x>1820</x>
      <y>610</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(221, 165, 51);
color: rgb(74, 37, 0);

font: 81 10pt &quot;SF Pro Rounded&quot;;
</string>
    </property>
    <property name="text">
     <string>Check</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>321</width>
      <height>1081</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(46, 79, 79);</string>
    </property>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>161</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>SF Pro Rounded</family>
       <pointsize>30</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(221, 165, 51);</string>
     </property>
     <property name="text">
      <string>Robot</string>
     </property>
    </widget>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>190</y>
       <width>301</width>
       <height>104</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>Roboto Condensed Light</family>
          <pointsize>16</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Time :</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="Date_Label">
        <property name="font">
         <font>
          <family>Roboto Condensed Light</family>
          <pointsize>13</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="Time_Label">
        <property name="font">
         <font>
          <family>Roboto Condensed Light</family>
          <pointsize>13</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>Roboto Condensed Light</family>
          <pointsize>16</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Date :</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="bt_startpage">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>400</y>
       <width>261</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>SF Pro Rounded</family>
       <pointsize>16</pointsize>
       <weight>10</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(221, 165, 51);
font: 81 16pt &quot;SF Pro Rounded&quot;;
</string>
     </property>
     <property name="text">
      <string>Start Page</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bt_viewalldata">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>520</y>
       <width>261</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>SF Pro Rounded</family>
       <pointsize>16</pointsize>
       <weight>10</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(221, 165, 51);


font: 81  &quot;SF Pro Rounded&quot;;
</string>
     </property>
     <property name="text">
      <string>Employee Data </string>
     </property>
    </widget>
    <widget class="QPushButton" name="bt_help">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>640</y>
       <width>261</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>SF Pro Rounded</family>
       <pointsize>16</pointsize>
       <weight>10</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(221, 165, 51);


font: 81 16pt &quot;SF Pro Rounded&quot;;
</string>
     </property>
     <property name="text">
      <string>Help</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bt_exit">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>880</y>
       <width>261</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>SF Pro Rounded</family>
       <pointsize>16</pointsize>
       <weight>10</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(221, 165, 51);


font: 81 16pt &quot;SF Pro Rounded&quot;;
</string>
     </property>
     <property name="text">
      <string>Exit</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bt_contactus">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>760</y>
       <width>261</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>SF Pro Rounded</family>
       <pointsize>16</pointsize>
       <weight>10</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(221, 165, 51);



font: 81 16pt &quot;SF Pro Rounded&quot;;
</string>
     </property>
     <property name="text">
      <string>Contact Us</string>
     </property>
    </widget>
   </widget>
   <zorder>widget</zorder>
   <zorder>label_5</zorder>
   <zorder>label_7</zorder>
   <zorder>bt_4</zorder>
   <zorder>bt_6</zorder>
   <zorder>bt_8</zorder>
   <zorder>bt_2</zorder>
   <zorder>bt_7</zorder>
   <zorder>bt_9</zorder>
   <zorder>bt_1</zorder>
   <zorder>bt_3</zorder>
   <zorder>bt_0</zorder>
   <zorder>bt_anpr</zorder>
   <zorder>bt_face</zorder>
   <zorder>groupBox</zorder>
   <zorder>imgLabel</zorder>
   <zorder>frameF</zorder>
   <zorder>groupBox_2</zorder>
   <zorder>bt_viewall_2</zorder>
   <zorder>frameP_2</zorder>
   <zorder>frameP</zorder>
   <zorder>bt_add</zorder>
   <zorder>bt_check</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
